K_deployment.md

```diff
@@ -8,6 +8,15 @@ Now that we've got our application prepared for production, getting it running i
 MIX_ENV=prod PORT=4001 iex -S mix phoenix.server
 ```
 
+To run the application in detached mode
```

Comment:
We should probably complete this sentence, and maybe give a little context. :)

How about something like this:
We can run our application detached from the iex console. This effectively daemonizes the process so it can run independently in the background. The command to do that looks like the following.